src/HomeComponent/HomeComponent.js

```diff
@@ -19,28 +20,31 @@ class HomeComponent extends Component {
   handleClick = () => {
     this.setState({ gif: this.state.list[Math.floor(Math.random() * this.state.list.length)]});
   }
-  
+
   componentDidMount() {
-    console.log('componentDidMount')
-    axios.get(`https://api.giphy.com/v1/gifs/search?api_key=${apikey}&q=cheeseburgers`).then((response) => {
+    // console.log('componentDidMount')
+    axios.get(`https://api.giphy.com/v1/gifs/search?api_key=${apikey}&q=cats`).then((response) => {
       console.log('response', response)
       const { data } = response.data;
       this.setState({ gif: data[Math.floor(Math.random() * data.length)], list: data });
     })
   }
-  // componentDidUpdate() {
```

Comment:
Remove all of this commented code, as you don't need it and it is increasing the number of lines of code you have to look through